extension/script/docs-rs.js

```diff
@@ -91,6 +91,11 @@ document.addEventListener("DOMContentLoaded", async () => {
         // Query installed crates from chrome.storage API
         let crates = await storage.getItem("crates") || {};
         let crate = crates[crateName];
+        if(!crate && crates[targetCrateName]) {
```

Comment:
How about:
```js
let crate = crates[crateName] || crates[targetCrateName];
```

Reply:
The creatName will be used here:
`await CrateDocManager.removeCrate(crateName)`